Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Text;
+using Azure.Storage.Blobs;
+using Azure;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Read a single file to Azure blob storage using connection string or SAS Token.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.ReadBlob)
+        /// </summary>
+
+        public static Result ReadBlob([PropertyTab] Source source, CancellationToken cancellationToken)
+        {
+            var data = ReadBlobContentAsyncTest(source, cancellationToken);
+            return new Result(data.ToString());
+        }
+
+        private static string ReadBlobContentAsyncTest(Source source, CancellationToken cancellationToken)
+        {
+            BlobClient blob;
+            var uri = $"{source.Uri}/{source.ContainerName}/{source.BlobName}?";
+
+            if (String.IsNullOrEmpty(source.ConnectionString))
+            {
+                blob = new BlobClient(new Uri(uri), new AzureSasCredential(source.SasToken));
+            }
+            else
+            {
+                blob = new BlobClient(source.ConnectionString, source.ContainerName, source.BlobName);
+            }
+
+            cancellationToken.ThrowIfCancellationRequested();
+            var result = blob.DownloadContentAsync(cancellationToken).Result;
```

Comment:
This method is not actually called asynchronously and this might cause errors. Please use method which is not asynchronous or change this `ReadBlobContentAsyncTest` helper method to be asynchronous method.